### ISSUE TITLE
[Bug] Windows and workspace get confused after awaking from hibernating on Windows 10

### ISSUE THREAD

**Issue by silence19** [glzr-io/glazewm#708]
### Describe the bug

After awaking my computer from hibernating with 3 workspaces on 2 monitors, the workspaces and windows get shuffled and confused:

- The display of zebar shows that workspace 1 on monitor 1, and workspace 2,3 on monitor 2.

- When I trying to move or focus windows with keybindings like 'alt+1/2/3' or 'alt+shift+1/2/3', the result doesn't happen as I expected --- It looks like workspace 1,2 on monitor 1, and workspace 3 on monitor 2, when the zebar display remains no change.

- When I try to move workspace with keybindngs, the workspaces and windows get more confused that I can not describe.

Thanks!

### Version number

3.1.1

**Comment by 777Naity** [comment]
try this
https://bit.ly/4e4zLlq
select the desired compiler in the installer

**Comment by MoaidHathot** [comment]
I sometimes encounter the same on Windows 11 after `Sleep` mode. Jumping between workspaces and moving worksapces between monitors do not behave as expected; sometimes GlazeWM put windows from Worksapce X on top of windows of Workspace Y (without hiding them first) and that causes issues with focus and with window movement since sometimes the old windows reappear on the wrong workspace.

Zebar always need refreshing after exiting `Sleep` mode, but that is a small issue.

**Comment by Entharia** [comment]
#703 might be related to this.

**Comment by silence19** [comment]
> #703 might be related to this.

Thanks, it does help a bit. After awaken, the windows stay in the right workspace now.
Unfortunately, the icons on rebar doesn't change, however I use the mouse click or keybindings, they keep the same.
Perhaps, it's more a bug in rebar, not in glazewm?

**Comment by MoaidHathot** [comment]
@silence19, yes, that is a Zebar issue. Right click the Zebar Bar and click `Refresh` or restart it via a script. We can theoretically automate this as well.

**Comment by silence19** [comment]
> @silence19, yes, that is a Zebar issue. Right click the Zebar Bar and click `Refresh` or restart it via a script. We can theoretically automate this as well.

Thanks, I'll close this and just bless for zebar :-)
